src/protocols/icd/Constants.h

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I don't beleive this is necessary since. For the check-in message,
- Protocol : SecureChannel
- MsgType : ICD_CheckIn

see src/protocols/secure_channel/Constants.h